lib/assets/javascripts/turbolinks.js.coffee

```diff
@@ -129,8 +129,9 @@ replace = (html, options = {}) ->
   triggerEvent EVENTS.LOAD, loadedNodes
 
 changePage = (doc, options) ->
-  [title, targetBody, csrfToken] = extractTitleAndBody(doc)
-  title ?= options.title
+  title = options.title
+  [extractedTitle, targetBody, csrfToken] = extractTitleAndBody(doc)
+  title ?= extractedTitle
```

Comment:
Sorry this is a little beyond my CoffeeScript knowledge. It does compile to something that looks legit